src/util/report.py

```diff
@@ -94,6 +94,8 @@ def html_escape(text):
 
 def add_to_migration_reports(message, category="Other", format="text"):
     assert format in {"text", "html", "md", "rst"}
+    if len(message) > 10000:
```

Comment:
I'd track instead the total size of the report. Once big enough (TBD how big) output a warning on each new report added.